### PR TITLE
Add longhorn components recovery

### PR DESCRIPTION
**Pull request by khushboo-rancher**
1. This is to include components recovery test in the release testing.
2. Removed air gap test from manual test as this is automated already.

Signed-off-by: Khushboo <fnu.khushboo@suse.com>

### CODE REVIEW

File: docs/content/manual/pre-release/resiliency/test-longhorn-component-recovery.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+1. Create a volume `vol-3` with backing image.
+1. Create an RWX volume `vol-4`.
+1. Write some data in all the volumes created and compute the md5sum.
+1. Have all the volumes in attached state.
+
+####Test steps:
+1. Delete the IM-e from every volume and make sure every volume recovers. Check the data as well.
+1. Start replica rebuilding for the aforementioned volumes, and delete the IM-e while it is rebuilding. Verify the recovered volumes.
+1. Delete the Share-manager pod and verify the RWX volume `vol-4` is able recover. Verify the data too.
+1. Delete the backing image pod and verify the pod gets recreated.
```
Comment (innobead) [inline docs/content/manual/pre-release/resiliency/test-longhorn-component-recovery.md:20]:
```suggestion
1. Delete the backing image manager pod and verify the pod gets recreated.
```

**Review by innobead — CHANGES_REQUESTED** [review]
LGTM, just one request to change.

File: docs/content/manual/pre-release/resiliency/test-longhorn-component-recovery.md (hunk 1 above)
Comment (khushboo-rancher) [reply docs/content/manual/pre-release/resiliency/test-longhorn-component-recovery.md]: Done.

**Review by innobead — APPROVED** [review]
This is good!

**Pull request merged**